youtube_dl/extractor/cbc.py

```diff
@@ -249,13 +280,19 @@ def _valid_device_token(self):
         return self._device_id and self._device_token
 
     def _register_device(self):
+        signature = self._signature()
         self._device_id = self._device_token = None
         result = self._download_xml(
             self._API_BASE_URL + 'device/register',
             None, 'Acquiring device token',
             data=b'<device><type>web</type></device>')
         self._device_id = xpath_text(result, 'deviceId', fatal=True)
         self._device_token = xpath_text(result, 'deviceToken', fatal=True)
+        if signature:
+            data = '<login><token>{}</token><device><deviceId>{}</deviceId><type>web</type></device></login>'.format(signature, self._device_id).encode()
```

Comment:
`{}` won't work in python 2.6.

Reply:
Thanks! I've updated this to use `{0}` and `{1}`